### PR TITLE
Added research on AuroraServerless

### PR DESCRIPTION
**Pull request by AndrewDorn**
## Summary of Changes

Please write a sentence or two summarizing the proposed change.

Addresses [ORCA-224: Research Spinup Period of Aurora Serverless](https://bugs.earthdata.nasa.gov/browse/ORCA-224)

## Changes

* Added research around Aurora Serverless.

## Type of change

- [x] This change requires a documentation update

## How Has This Been Tested?

Solutions tested against Aurora-Serverless using SQLAlchemy Python code.

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the necessary documentation (remove those that do not apply)
    - [x] User documentation
- [x] My changes generate no new warnings
- [x] Any dependent changes have been merged and published in downstream modules
- [x] My code has passed security scanning
    - [x] Snyk
    - [x] git-secrets

### CODE REVIEW

File: website/docs/developer/research/research-AuroraServerless.md [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+- When scaling occurs, connections are cancelled, transactions are rolled back, locks are dropped, and temporary tables are dropped.
+  :::note
+  In our current technology, PostgreSQL, [each query is a transaction](https://www.postgresql.org/docs/8.3/tutorial-transactions.html). 
+  :::
+- When the Aurora DB is scaled down to zero, it takes some time to scale up.
+  - With SQLAlchemy this translated to a 30-40 second hang on ```engine.begin()```
+  - The default timeout for SQLAlchemy appears to be [30 seconds](https://docs.sqlalchemy.org/en/14/core/engines.html), though as mentioned previously connecting took 40 seconds with no issues.
+    May be multiple timeouts in play.
+  - Scaling up after this point did not cause query performance degradation, despite what was documented. However, the UI is poorly made, and may be reporting incorrectly.
+- Connections canceled due to scaling raise an error in SQLAlchemy.
```
Comment (rizbi-hassan) [inline website/docs/developer/research/research-AuroraServerless.md:30]: Are there any logs that show up on cloudwatch regarding this?

Comment (rizbi-hassan) [reply website/docs/developer/research/research-AuroraServerless.md:30]: Cloudwatch does not get notified according to further research from Andrew.

**Review by rizbi-hassan — APPROVED** [review]

**Comment by Quarriers89** [comment]
@AndrewDorn PR #120 looks good to me.

File: website/docs/developer/research/research-AuroraServerless.md [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+  (Background on this error at: https://sqlalche.me/e/14/e3q8)
+  ```
+  
+  - Retrying when these errors occur should be sufficient to sidestep scaling errors, as scaling does not take long.
+  :::warning
+    Disconnections caused by these errors are NOT reported to Cloudwatch. Connecting lambdas/modules should have reliable and robust error logging.
+  :::
+
+### Future Direction
+- Modifications should be made to database code to account for AdminShutdown, OperationalError, and connection timeout issues.
```
Comment (bhazuka) [inline website/docs/developer/research/research-AuroraServerless.md:87]: Did you want to speak to performing a potential check query like Cumulus does or another probe query to wake things up or is managing errors and timeouts with retries sufficient enough to overcome any errors or issues?

**Review by bhazuka — COMMENTED** [review]
Looks good to me. Just a few minor questions.

File: website/docs/developer/research/research-AuroraServerless.md (hunk 2 above)
Comment (AndrewDorn) [reply website/docs/developer/research/research-AuroraServerless.md:87]: I'd prefer the later. Their strategy of a test query works for checking for spin-up, but doesn't cover errors caused by scaling, so we'd need error-handling logic at each step anyway. It also doubles time spent waiting on network latency, when we would just send our actual query and be done with it.

**Review by bhazuka — APPROVED** [review]
Looks good

**Pull request merged**